components/playback/controls/StimulationStudioPlateWell.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <div>
+    <div
+      class="div__simulationstudio-plate-well-location"
+      :style="'top:' + computed_top + 'px;' + 'left:' + computed_left + 'px;'"
+    >
+      <PlateWell
+        :classname="classname"
+        :svg_height="svg_height"
+        :svg_width="svg_width"
+        :circle_x="circle_x"
+        :circle_y="circle_y"
+        :radius="radius"
+        :strk="strk"
+        :plate_fill="protocol_fill"
+        :stroke_wdth="stroke_wdth"
+        :index="index"
+      ></PlateWell>
+    </div>
+    <span
+      class="span__simulationstudio-plate-well-protocol-location"
+      :style="
+        'top:' +
+        computed_protocol_top +
+        'px;' +
+        'left:' +
+        computed_protocol_left +
+        'px;'
+      "
+    >
+      {{ protocol_type }}
+    </span>
+  </div>
+</template>
+<script>
+import PlateWell from "@/components/playback/controls/PlateWell.vue";
+
+export default {
+  name: "StimulationStudioPlateWell",
+  components: {
+    PlateWell,
+  },
+  props: {
```

Comment:
`stroke` is a css property type for SVG, a direct utilization matching the name `stroke` in the Vuejs component PlateWell results in css property of the component to be modified, but our business logic is to pass the value of the `stroke` to be applied on an svg element <circle>, so I have just retained the same reference as `strk`, the same applies in case of the attribute of `stroke-width` for the `stroke_wdth` on referring the same name it results in replacement at higher element.

We cannot match the names exactly the same as it results in loosing the flexible re-use of the widget.

Reply:
could you say `stroke_color` instead of `strk` then? it can be very confusing for someone to try to figure out what a variable name is if all the vowels have been removed from the word

how would `stroke_width` (with an underscore for the prop name) conflict with `stroke-width` (with a hyphen, which is the CSS property)?

Reply:
`PlateWell` is an atomic widget, modifying on that component will create a large amount of files which are built on top of that.

Why didn't you find it different when PlateWell was reviewed ? 

This cannot be done it will create lots of changes.

Reply:
I'm not asking you to modify `PlateWell` right now, just `StimulationPlateWell` to stop the issue from cascading further in the library.

Human reviewers often miss things the first time reviewing something.  That is why we use automated linting and other tools for as much as possible to keep our code clean, readable and following best practices. It is well documented that human reviewers can also miss bugs, which is why proper automated testing and Test Driven Development is another important aspect of maintaining a high quality codebase.

Reply:
ok... will work on the same.